starwars/src/components/MainContainer.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import InfoContainer from './info/InfoContainer';
+import './main.css';
+
+const MainContainer = (props) => {
+    return (
+        <div className="main-container">
+            <h1>React Wars</h1>
+            <img src={require("../Luke.jpeg")} alt=""/>
```

Comment:
Images in React are difficult, but you can generally refer to them by path if you put them in your public folder. so public/img could be referred to as `./img/<whatever your img url is>`